src/App.css

```diff
@@ -1,42 +1,56 @@
 #root {
-  max-width: 1280px;
+  width: 100vw;
   margin: 0 auto;
-  padding: 2rem;
-  text-align: center;
+  padding-left:2rem;
+  padding-right:2rem;
 }
 
-.logo {
-  height: 6em;
-  padding: 1.5em;
-  will-change: filter;
-  transition: filter 300ms;
-}
-.logo:hover {
-  filter: drop-shadow(0 0 2em #646cffaa);
+
+
+
+.header{
+  text-align: left;
+  background-color: rgb(148, 204, 148);
 }
-.logo.react:hover {
-  filter: drop-shadow(0 0 2em #61dafbaa);
+
+.nav{
+  text-align: left;
 }
 
-@keyframes logo-spin {
-  from {
-    transform: rotate(0deg);
-  }
-  to {
-    transform: rotate(360deg);
-  }
+li{
+  list-style: none;
+  margin-left: 0;
+  margin-left: -40px;
 }
 
-@media (prefers-reduced-motion: no-preference) {
-  a:nth-of-type(2) .logo {
-    animation: logo-spin infinite 20s linear;
-  }
+.article-list{
+  display: flex;
+  flex-direction: column;
+  
+  text-align: left;
+  font-size: 0.7rem;
+  
+  border: 2px solid red;
+  padding-left: 2rem;
+  padding-right: 2rem;
 }
 
-.card {
-  padding: 2em;
+.article-card{
+  text-align: left;
+  border: 1px solid black;
+  padding-left: 1rem;
+  
+  margin-top: 0.5rem;
+  background-color:rgb(245, 245, 245);
 }
 
-.read-the-docs {
-  color: #888;
+
+.article-card h3{
+  font-size: 0.8rem;
+  text-decoration: underline;
+  text-transform: capitalize;
 }
+
+.article-card img{
+  width:7rem;
+}
```

Comment:
Currently when I open the article list, I can only see the images. All the text (like title) are white so they are not visible. A colour for that might need to be specified somewhere in your .css file.